src/istio/control/http/attributes_builder_test.cc

```diff
@@ -342,6 +342,18 @@ attributes {
     string_value: "NR"
   }
 }
+attributes {
+  key: "rbac.shadow.response_code"
+  value {
+    string_value: "403"
+  }
+}
+attributes {
+  key: "rbac.shadow.effective_policy_id"
```

Comment:
Can you pick some better name? Shadow is not descriptive enough.

Reply:
cc @mandarjog for attribute name suggestions...

Reply:
renamed to ```permissive```(envoy uses ```shadow```, istio uses ```permissive``` )